lib/runner/TestRunner.js

```diff
@@ -108,7 +109,11 @@ module.exports = class TestRunner {
             if (test.hasGoto || !(invoker.batchSupported() && batchEnabled)) {
                 results = await this.sequentialRun(invoker, testSuiteWithLocale, test.interactions, context);
             } else {
-                results = await this.batchRun(invoker, testSuiteWithLocale, test.interactions, context);
+                if (invoker.batchSupported() && batchEnabled && asyncMode) {
+                    results = await this.sequentialRun(invoker, testSuiteWithLocale, test.interactions, context);
```

Comment:
should not batchRun here

Reply:
we need to do the sequential run for batch process in async mode in order to scan de responses as we get them from the conversation endpoint.